c/build_all/linux/build.sh

```diff
@@ -65,6 +66,7 @@ process_args ()
               "--no-http" ) build_http=OFF;;
               "--no-mqtt" ) build_mqtt=OFF;;
               "--build-python" ) build_python=ON;;
+              "--python-3" ) python_3=ON;;
```

Comment:
lets investigate a --build-python x.y build option here. We need to support a variety of py version